bluesky/callbacks/best_effort.py

```diff
@@ -441,6 +441,53 @@ def clear(self):
         self._buffer = StringIO()
         self._baseline_toggle = True
 
+    def plot_prune_fifo(self, num_lines, y_signal, x_signal):
+        """
+        Find the plot with axes x_signal and y_signal.  Replot with only the last *num_lines* lines.
+
+        .. note:: This is not a bluesky plan.  Call it as a normal Python function.
+
+        Example to remove all scans but the last:
+        >>> bec.plot_prune_fifo(1, noisy, m1)
+
+        Parameters
+        ----------
+        num_lines: int
+            number of lines (plotted scans) to keep, must be >= 0
+        y_signal: object
+            instance of ophyd.Signal (or subclass), 
+            dependent (y) axis
+        x_signal: object
+            instance of ophyd.Signal (or subclass), 
+            independent (x) axis
+
```

Comment:
```suggestion
```

Reply:
Doesn't Sphinx need this blank line due to the indented content?

Reply:
Not after the list of arguments in case we don't have the continuation of the docstring, as far as I know.